lib/config-validator.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
```

Comment:
This is removed as we are going to use JSON validation instead of this. It was supposed to be removed before updating the pull request, but I missed it.